$DragonFly: src/gnu/usr.bin/groff/tmac/patches/groff_mdoc.man.patch,v 1.1 2006/03/08 23:09:30 swildner Exp $

--- ../../../../../contrib/groff-1.19/tmac/groff_mdoc.man	2005-09-09 16:10:41.000000000 +0200
+++ groff_mdoc.man	2006-03-08 23:57:58.000000000 +0100
@@ -183,6 +183,7 @@
 .    It "BSD Macro"
 .    It "NetBSD Macro"
 .    It "FreeBSD Macro"
+.    It "DragonFly Macro"
 .    It "OpenBSD Macro"
 .    It "BSD/OS Macro"
 .    It "UNIX Macro"
@@ -869,6 +870,8 @@
 2.2.2, 2.2.5, 2.2.6, 2.2.7, 2.2.8, 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 4.0, 4.1,
 4.1.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.6.2, 4.7, 4.8, 4.9, 4.10, 5.0, 5.1, 5.2,
 5.2.1, 5.3
+.It DragonFly
+1.0, 1.1, 1.2, 1.3, 1.4, 1.5
 .El
 .Ed
 .Pp
@@ -2167,6 +2170,25 @@
 command above in section
 .Sx "TITLE MACROS" .
 .
+.Ss "DragonFly Macro"
+.
+.Pp
+.Dl Usage: .Dx Oo Ao version Ac Oc ...
+.Pp
+.Bl -tag -width ".Li .Dx\ 1.4\ ." -compact -offset 15n
+.It Li .Dx
+.Dx
+.It Li ".Dx 1.4 ."
+.Dx 1.4 .
+.El
+.Pp
+For possible values of
+.Ao version Ac
+see the description of the
+.Ql .Os
+command above in section
+.Sx "TITLE MACROS" .
+.
 .Ss "OpenBSD Macro"
 .
 .Pp
